api/envoy/api/v2/core/grpc_service.proto

```diff
@@ -53,6 +53,11 @@ message GrpcService {
 
         // https://grpc.io/grpc/cpp/namespacegrpc.html#a6beb3ac70ff94bd2ebbd89b8f21d1f61
         google.protobuf.Empty google_default = 2;
+
+        // Use local channel credentials. Currently only UDS is supported.
```

Comment:
It only works if target_url is prefixed with "unix:"
Certain Google gRPC features, such as passing Google default call credential, only works with a valid channel credential.  Local credential is a valid channel credential.